components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIGatewayManager.java

```diff
@@ -679,6 +695,9 @@ private void addEndpoints(API api, APITemplateBuilder builder, GatewayAPIDTO gat
         if (api.getGatewayLabels() != null) {
             for (Label label : api.getGatewayLabels()) {
                 Environment environment = getEnvironmentFromLabel(label);
+                if (debugEnabled) {
+                    log.debug(api.getId() + "removing from the environment of " + environment);
```

Comment:
Since this is a lable, shall we change the debug message to the label. Also can we just print the environment? This is the environment object?

Reply:
Also, you might need to add a space between apiID and debug message.

Reply:
https://github.com/wso2/carbon-apimgt/pull/9019/commits/29a2e4bba591e33476f9fcce64bd8668945123c4